lib/mailchimp_api/resources/campaign.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module MailchimpAPI
+  class Campaign < Base
+    extend MailchimpAPI::Support::Countable
+
+    include MailchimpAPI::Support::PatchUpdate
+
+    self.collection_parser = CollectionParsers::Campaign
+
+    # Define all bodiless, non-RESTful actions
+    %w[
+      cancel-send
+      create-resend
+      pause
+      replicate
+      resume
+      unschedule
+    ].each do |path_name|
+      define_method path_name.sub('-', '_') do
+        path = element_path(prefix_options) + "/actions/#{path_name}"
+        connection.post path, nil, self.class.headers
+      end
+    end
+
+    # Calls the '/send' endpoint, but we obviously can't override the Ruby 'send' method.
+    def send_campaign
+      path = element_path(prefix_options) + '/actions/send'
+      connection.post path, nil, self.class.headers
+    end
+
+    def schedule(time, use_timewarp = false, batch_delivery = false)
```

Comment:
I'm always on the fence if we should preload errors on behalf of the API. For example, should this explode if time isn't `> Time.now`?

Reply:
My view is that we are a passthrough to the API, not that we enforce the API's behaviour. My view may be wrong, of course, but it's definitely more work for us to catch errors that the API will return anyway. Especially when the API's behaviour might change.

Reply:
Good enough for me